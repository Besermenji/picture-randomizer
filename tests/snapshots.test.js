import initStoryshots from '@storybook/addon-storyshots';
import { mockRandomForEach } from 'jest-mock-random';

mockRandomForEach(0.5);
initStoryshots();
