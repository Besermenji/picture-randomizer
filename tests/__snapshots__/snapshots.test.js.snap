// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots PictureRandomizer custom css 1`] = `
<img
  alt="bro waving"
  onClick={[Function]}
  src="https://media.giphy.com/media/2yxkPj5mWYOGh4UYBs/giphy.gif"
  style={
    Object {
      "borderRadius": "50%",
      "height": 200,
      "width": 200,
    }
  }
/>
`;

exports[`Storyshots PictureRandomizer with images array 1`] = `
<img
  alt="bro waving"
  onClick={[Function]}
  src="https://media.giphy.com/media/2yxkPj5mWYOGh4UYBs/giphy.gif"
  style={Object {}}
/>
`;

exports[`Storyshots PictureRandomizer without props 1`] = `null`;
